LayoutTests/http/tests/media/track/toggle-show-subtitles-context-menu-item.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Test show/hide subtitles context menu item behavior</title>
+        <script src="../../../../resources/js-test.js"></script>
+        <script src="../../../../resources/js-test-pre.js"></script>
+        <script src="../../media-resources/video-test.js"></script>
+        <script src="../../media-resources/media-file.js"></script>
+        <script>
+            async function runTest()
+            {
+                run("internals.setUserPreferredLanguages(['en'])");
+                run("internals.setPrimaryAudioTrackLanguageOverride('ru')");
+                run("internals.settings.setShouldDisplayTrackKind('Captions', true)");
+                video = document.getElementById('video');
+                video.src = findMediaFile('video', '../resources/test');
+                await waitFor(video, 'canplay');
+                let track = document.createElement('track');
+                track.src = '../resources/hls/subtitles/subtitles.vtt';
+                track.setAttribute('srclang', 'en');
+                track.setAttribute('kind', 'captions');
+                video.appendChild(track);
+                await waitFor(video.textTracks, 'addtrack');
+
+                testExpected("video.textTracks[0].mode", 'disabled');
+                x = video.offsetLeft + video.offsetWidth / 2;
+                y = video.offsetTop + video.offsetHeight / 2;
+                eventSender.mouseMoveTo(x,y);
+                eventSender.contextClick().find(item => item.title === "Show Subtitles (auto)").click();
```

Comment:
It may be worth putting the result of `find(item => item.title === "Show Subtitles (auto)")` into variable so you can test for its presence separately, so the cause of the failure if it is somehow not in the menu will be more obvious.

Reply:
Good idea!